ultet/src/main/scala/za/co/absa/ultet/dbitems/DBFunction.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package za.co.absa.ultet.dbitems
+
+trait DBFunction extends DBItem {
+def fnName: String
+  def paramTypes: Seq[String]
```

Comment:
argument names will be needed too, and also output type(s);

I created something similar, perhaps we can merge them into one somewhere? https://github.com/AbsaOSS/fa-db/pull/75/files#diff-c3121d472cb3b94fccd4d0c299a444f9b596c5d81c77bed13aa1e1458fe1d4f5

Reply:
Well, that depends - if you choose to recreate the SQL, then yes, otherwise, if you keep it as-is, you don't need it for signature comparison - only types are compared, I believe.

Reply:
Nope not needed. Just the types should be enough.